addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -97,13 +95,27 @@ publicWidget.registry.RatingPopupComposer = publicWidget.Widget.extend({
 
     /**
      * @private
-     * @param {OdooEvent} ev
+     * @param {OdooEvent} event
      */
-    _onReloadRatingPopupComposer: function (ev) {
-        this._reloadRatingPopupComposer(ev.data);
+    _onReloadRatingPopupComposer: function (event) {
+        const data = event.data;
+
+        // Refresh the internal state of the widget
+        this.rating_avg = data.rating_avg;
```

Comment:
Does data contain other keys that do not go into the options?
Otherwise, we would use an "extend" to simplify.

`this.options = _.extend(this.options, data);`